### ISSUE TITLE
Ctrl + Shift + S not working

### ISSUE THREAD

**Issue by luciarodriguez** [Infragistics/livewire#109]
Trying to "Save As" is not working for me. This is with the latest version (1.1005)

**Comment by craigshoemaker** [comment]
@luciarodriguez does anything happen? Do you get an error?

**Comment by luciarodriguez** [comment]
No, I don't get any errors. I just click the button and nothing happens. I've attached a video.

[LiveWire video.zip](https://github.com/Infragistics/livewire/files/777131/LiveWire.video.zip)
